.github/workflows/people.yml

```diff
@@ -71,34 +71,76 @@ jobs:
         with:
           node-version: '18.x'
           cache: 'yarn'
-          cache-dependency-path: src/frontend/app/desk/yarn.lock
+          cache-dependency-path: src/frontend/yarn.lock
       - name: Install dependencies
         run: yarn install --frozen-lockfile
       - name: Test Desk App
         env:
           CI: true
         run: yarn test
 
-  build-front-desk:
+  test-e2e:
     runs-on: ubuntu-latest
-    env:
-      NEXT_PUBLIC_API_URL: http://localhost:8071/api/v1.0/
-    defaults:
-      run:
-        working-directory: src/frontend/app/desk
+    
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        
+      - name: Set services env variables
+        run: |
+          cp env.d/development/common.dist env.d/development/common
+          cp env.d/development/postgresql.dist env.d/development/postgresql
+          cp env.d/development/kc_postgresql.dist env.d/development/kc_postgresql
+          cp env.d/development/crowdin.dist env.d/development/crowdin
+
+      - name: Start Docker Servers
+        run: |
+          # Check if Docker Layer Caching is available
+          if [[ -z "${{ runner.docker }}" ]]; then
+            echo "Docker Layer Caching is not available. Building without caching."
+          else
+            # Will restore the cached Docker layers, if available
+            docker-compose pull --ignore-pull-failures
+            docker-compose build --pull
+          fi
+      
+          # Start the services needed for the tests
+          docker-compose up -d nginx
+          docker-compose up -d app-dev
+          docker-compose up -d celery-dev
+          docker-compose up -d keycloak
```

Comment:
same, could we do a `make run` ?

Reply:
[2c94b4c](https://github.com/numerique-gouv/people/pull/13/commits/2c94b4c4f2554d25ffafbe388777fb72c1b079bc)

After I am not 100% comfortable with that, `make run` could start things that we don't need with the CI, making our job slower. Let's try to remember it.

Reply:
Yes, but at this stage, if it doesn't slow down the CI, it should not be an issue